pkg/server/http/server.go

```
@@ -22,7 +26,13 @@ func Server(opts ...Option) (http.Service, error) {
 		http.Flags(options.Flags...),
 	)
 
+	as, err := getAccountsService()
+	if err != nil {
+		return service, err
```

shouldn't it be `return nil, err`? Or are we actually ignoring this error?